test/generateTasks.spec.js

```diff
@@ -77,8 +88,21 @@ describe('generateTasks', () => {
     })
   })
 
+  it('should not match non-children files', () => {
+    const relPath = path.resolve(path.join(process.cwd(), '..'))
+    resolveGitDir.mockReturnValueOnce(relPath)
```

Comment:
@okonet check for filtering out files that aren't in the `cwd` but show up because `gitDir` is diff